workflow/scripts/postprocess_go_enrichment.py

```diff
@@ -0,0 +1,39 @@
+import pandas as pd
+from ast import literal_eval as make_tuple
+
+
+def calculate_enrichment(ratio_in_study_str, ratio_in_pop_str):
+    """Calculate enrichment based on ratios provided as strings."""
+    ratio_in_study = make_tuple(ratio_in_study_str)
+    ratio_in_pop = make_tuple(ratio_in_pop_str)
+    enrichment_study = ratio_in_study[0] / ratio_in_study[1]
+    enrichment_pop = ratio_in_pop[0] / ratio_in_pop[1]
+    return enrichment_study, enrichment_pop
```

Comment:
Wouldn't a function that works on a single tuple, and which returns a single value, make sense here? You could simply call it on the two tuples separately, as the input tuples are handled completely separately anwyays.